styles.css

```diff
@@ -0,0 +1,144 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+body {
+  font-family: Poppins, sans-serif;
+  margin: 0;
+}
+
+.section {
+  background-image: url(/assets/header-bg.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.container {
+  max-width: 80rem;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  overflow: auto; /* this allows elements to overflow out of the bounding box */
+}
+
+.container-small {
+  max-width: 57.5rem;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-container {
+  max-width: 80rem;
+  margin: 0 auto;
+
+  /* to centre the div */
+  padding: 1rem 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 0.5rem;
+
+  /* equal to 8px */
+}
+
+.logo {
+  padding: 10px 12px;
+
+  /* to be converted to REM */
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.nav-link {
+  padding: 10px 12px;
+
+  /* to be converted to REM */
+  margin: 0 0.5rem 0 0;
+  color: #344563;
+  font-size: 1rem;
+  font-weight: 500;
+}
+
+h1.heading {
+  font-size: 3rem;
+
+  /* equal to 48px */
+  font-weight: 700;
+}
+
+p.supporting-text {
+  font-size: 1.25rem;
+  font-weight: 400;
+  color: #344563;
+}
+
+.toolbar {
+  display: flex;
+}
+
+.call-to-action {
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  justify-content: space-between;
+  margin: 2rem 0 1rem 0;
+}
+
+.icons {
+  display: flex;
+}
+
+.social-icon {
+  margin-right: 1rem;
+}
+
+.headline-section {
+  padding: 10rem 0;
+}
+
+.twitter {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.github {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+/**
+.logo {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;}
+
+.about {
+  display: flex;
+  text-align: right;
+  text-align: right;
+  justify-content: space-between;
+}
+
+.contact {
+  display: flex;
+  text-align: right;
+  justify-content: space-between;
+}
+
+.port {
+  display: flex;
+  text-align: right;
+  justify-content: space-between;
```

Comment:
kindly remote this comment code to make your file clean and professional

Reply:
I removed this, thank you @vic778